app/views/lecturers/new.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>New Lecturer</h1>
+
```

Comment:
We dont need views for lecturers. the info for lecturers will be presented with the associated resource, subjects